### PR TITLE
Freeze issue

### PR DESCRIPTION
**Pull request by soumyanishan**
made the freeze part sequential instead of threads

### CODE REVIEW

File: VMBackup/main/snapshotter.py [hunk 1]
```diff
@@ -92,8 +92,8 @@ def snapshot(self, sasuri, meta_data,snapshot_result_error,global_logger,global_
         temp_logger=temp_logger + ' snapshot ends..'
         global_logger.put(temp_logger)
         global_error_logger.put(error_logger)
-        if(snapshot_error.errorcode != CommonVariables.success):
-            snapshot_result_error.put(snapshot_error)
+        #if(snapshot_error.errorcode != CommonVariables.success):
```
Comment (ahmetalpbalkan) [inline VMBackup/main/snapshotter.py]: please remove the code line if unused

File: VMBackup/main/handle.py [hunk 2]
```diff
@@ -165,24 +165,24 @@ def snapshot():
         backup_logger.log(errMsg, False, 'Error')
     snapshot_done = True
 def daemon():
-    global MyPatching,backup_logger,hutil,run_result,run_status,error_msg,freezer,para_parser
+    global MyPatching,backup_logger,hutil,run_result,run_status,error_msg,freezer,para_parser,snapshot_done
```
Comment (ahmetalpbalkan) [inline VMBackup/main/handle.py]: spaces between those would be more useful. 

have you tried running pep8 on this file?

Comment (soumyanishan) [inline VMBackup/main/handle.py]: @ahmetalpbalkan added the space. on running pep8 there are lot of spacing issue which we will take up while refactoring the VMBackup extension code

File: VMBackup/main/snapshotter.py (hunk 1 above)
Comment (soumyanishan) [inline VMBackup/main/snapshotter.py]: @ahmetalpbalkan done

**Comment by vityagi** [comment]
Looks good to me for merging to the master.

**Pull request merged**